examples/pytorch/MNIST/example1/mnist_autolog_example1.py

```diff
@@ -245,7 +264,8 @@ def optimizer_step(
     mlflow.set_tracking_uri(dict_args['tracking_uri'])
 
     model = LightningMNISTClassifier(**dict_args)
-    early_stopping = EarlyStopping(monitor="val_loss", mode="min", verbose=True)
+    early_stopping = EarlyStopping(monitor=dict_args["es_monitor"], mode=dict_args["es_mode"],
```

Comment:
Please verify there are no BC with PTL 1.0

Reply:
There is no change in `EarlyStopping` class. In PL 1.0 `early_stop_callback` arugment is removed